69986 do-po - Remove program and references

tyndmar opened the pull request:
Verified no references to do-po.p
Deleted file do-po.p

Commit:
69986 do-po - Remove program and references

Verified no references to do-po.p
Deleted file do-po.p

brad-advantzware reviewed (changes requested):
Mark - Grep still results in finds after this.  Callers to this are commented out but please remove commented out code so that a grep on do-po.p results in nothing

Commit:
69886 do-po Remove program and references

Programs changed (removed commented references to do-po)
oe/d-oeitem.w
oe/v-ordl.w
po/doPo.p
